components/homepage/whyMinted.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from "next/image";
+
+export default function WhyMinted() {
+    return (
+      <div id="WhyMinted" className="w-full bg-maroon/20 text-center 2xl:pt-20 pt-16 overflow-hidden">
+        <div className="2xl:text-6xl text-5xl font-semibold text-darkmaroon 2xl:mb-10 mb-8">
+          Why Minted
+        </div>
+        <div className="text-gray-600 2xl:text-xl text-lg w-[55rem] mx-auto flex flex-col space-y-8">
+          <div>
+            Minted is a feature rich application that connects donors and
+            non-profits in a seamless and efficient way. This app is dedicated
+            to help boost community projects by helping non-profits get the
+            funding they need. All features are user friendly with an easy to
```

Comment:
"feature-rich" instead of feature rich
"user-friendly" instead of user friendly (to match About content)